paddle/fluid/operators/controlflow/while_op_helper.h

```diff
@@ -40,5 +42,21 @@ void PrepareSafeEagerDeletionOnWhileOpAndWhileGradOp(
     const std::vector<framework::OperatorBase *> &while_ops,
     const std::vector<framework::OperatorBase *> &while_grad_ops);
 
+inline bool GetCondData(const framework::LoDTensor &cond) {
```

Comment:
Had better to put declaration at .h file and implementation at .cc file. Not inline it.

Reply:
Done.